aura/aura.cabal

```diff
@@ -97,6 +97,7 @@ library
     , network-uri       ^>=2.6
     , stm               ^>=2.5
     , time              >=1.8 && <1.10
+    , xdg-basedir       ==0.2.2
```

Comment:
Despite not being updated since 2012, this is actually in Stackage. Neat. 

As for how to actually pin it, let's do `^>= 0.2`.